### ISSUE TITLE
[Accessibility][Breadcrumb][Sample CSS] Right chevron icons CSS "content" property value is read by NVDA 

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1672]
**Steps to Reproduce**
1. Enable NVDA.
2. Read through breadcrumb with down arrow.

**Current Behavior**
Text alternatives for decorative icons in buttons are provided through the CSS `content` property, which screen reader users may or may not hear announced. 

Text content in the CSS `content` property is supported by modern Chrome and Firefox browsers and JAWS and NVDA versions, but is not supported in IE with either screen reader. Announcements will vary from one assistive technology/browser combination to another. 

**Expected behavior/code**
If an icon is purely decorative, it should be hidden from assistive technology by using an empty CSS `content` property value.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
1.1.1 Non-text Content

**Code snippet**
```
.t-cmp-clean .cmp-breadcrumb__item:after {
    content: 'chevron_right';
   ...
}
```

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
This bug is not specific to the Core Components, but to the CSS of the Component Library.
